docs/topics/Opcodes_and_Status_Codes.md

```diff
@@ -84,9 +84,9 @@ Our API will return semantically valid HTTP response codes based on the success
 
 | Code                      | Meaning                                                                              |
 |---------------------------|--------------------------------------------------------------------------------------|
-| 200 (OK)                  | The request completed successfully.                                                  |
+| 200 (OK)                  | The request was completed successfully.                                                  |
 | 201 (CREATED)             | The entity was created successfully.                                                 |
-| 204 (NO CONTENT)          | The request completed successfully but returned no content.                          |
+| 204 (NO CONTENT)          | The request was completed successfully but returned no content.                          |
```

Comment:
i'm not sure if this is correct or incorrect, but it isn't needed

Reply:
This is more consistent with 201, and it is correct as far as I know.